crates/virtio-queue/src/lib.rs

```diff
@@ -12,9 +12,10 @@
 
 //! A crate that exposes the virtio queue API.
 
-#![deny(missing_docs)]
+// #![deny(missing_docs)]
```

Comment:
Is this commented out because the new module has public objects, methods, etc. which don't have doc comments?